ReleaseNotes/RN230316.md

```diff
@@ -0,0 +1,66 @@
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_SQLServer_AuthN_Dont_Allow_Public_Network_Access](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#41-Azure_SQLServer_AuthN_Dont_Allow_Public_Network_Access)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_SQLDatabase_AuthZ_Use_AAD_Only](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#39-Azure_SQLDatabase_AuthZ_Use_AAD_Only)
+ * Added [Bulk Remediation Script](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1) for the control [Azure_AutomationAccounts_DP_Encrypt_Variables](https://github.com/azsk/AzTS-docs/blob/main/Scripts/RemediationScripts/Readme.md#40-Azure_AutomationAccounts_DP_Encrypt_Variables)
+
+## Tenant Scanner Framework Updates
+* Updated the target framework for all AzTS components to **net6.0** and function runtime version to **V4** in the entire AzTS solution as netcore3.1 is out of support.
+
+
+## Other Improvements
+* **AzTS UI** : Added new feature - Control remediation using scripts download from AzTS UI.
+*  **AzTS UI** :  Introduced a single side panel view on clicking over control name to display control and control evaluation related details.
+* **Policy** : Added new [Policy](https://github.com/azsk/AzTS-docs/blob/main/Policies/Readme.md) for following controls.
+    *  [Azure_APIManagement_AuthN_Disable_Management_API](https://github.com/azsk/AzTS-docs/blob/main/Policies/APIManagement/Azure_APIManagement_AuthN_Disable_Management_API/Readme.md)
+    *  [Azure_KubernetesService_AuthN_Enabled_AAD](https://github.com/azsk/AzTS-docs/blob/main/Policies/KubernetesService/Azure_KubernetesService_AuthN_Enabled_AAD/Readme.md)
+    *  [Azure_Subscription_Config_MDC_Setup_SecurityContacts](https://github.com/azsk/AzTS-docs/blob/main/Policies/Subscription/Azure_Subscription_Config_MDC_Setup_SecurityContacts/Readme.md)
+    *  [Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN](https://github.com/azsk/AzTS-docs/blob/main/Policies/APIManagement/Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN/Readme.md)
+    *  [Azure_KeyVault_NetSec_Disable_Public_Network_Access](https://github.com/azsk/AzTS-docs/blob/main/Policies/KeyVault/Azure_KeyVault_NetSec_Disable_Public_Network_Access/Readme.md)
+    *  [Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server](https://github.com/azsk/AzTS-docs/blob/main/Policies/SQLServer/Azure_SQLDatabase_Audit_Enable_Threat_Detection_Server/Readme.md)
+
+* **Policy** : Updated [Policy](https://github.com/azsk/AzTS-docs/blob/main/Policies/Readme.md) for following controls .
```

Comment:
Updated audit policies for the following security controls.

Reply:
Done